product_template_tags/README.rst

```diff
@@ -0,0 +1,71 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================
+Product Template Tags
+=====================
+
+This addon allow to add tags on products.
+
+Installation
+============
+
+Just install it!
+
+Configuration
+=============
+
+* Go on the product form view.
+* Create and edit tags using the form view
+
+.. figure:: path/to/local/image.png
+   :alt: alternative description
+   :width: 600 px
```

Comment:
remove

Reply:
alternatively, you can add a screen shot to display product form view with the new field. (not a blocking point)